content/en/network_monitoring/devices/network_topology_map.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Network Topology Map
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/network_monitoring/devices/data"
+  tag: "Documentation"
+  text: "Data Collected with Network Device Monitoring"
+- link: "https://www.datadoghq.com/blog/monitor-snmp-with-datadog/"
+  tag: "Blog"
+  text: "Monitor SNMP with Datadog"
+---
+
+<div class="alert alert-warning">The Network Topology Map for Datadog Network Device Monitoring is in public beta.</div>
+
+## Overview
+
+The Network Topology map provides an overview of your network's physical connections, so you can more easily identify issues in your devices and understand their upstream and downstream impacts. 
+
+{{< img src="/network_device_monitoring/network_topology_map/network_topology_map_search.mp4" alt="The network topology map, with vendor:cisco added to the search bar, and then the Filter nodes box filtered by nyc. A node is selected and the inspect option chosen, showing its connected nodes. One of the connected nodes is then selected, and the inspect option is selected again, displaying additional connected nodes" video="true" >}}
+
+## Setup
+
+The Datadog Agent version 7.46 and later automatically collects topology data. No additional installation is necessary.
+
+### Prerequisites
+
+1. LLDP is enabled on the device with LLDP data exposed through SNMP.
```

Comment:
Is LLDP general knowledge or would it be helpful to users to spell out the first instance of this?